client/src/component/App.jsx

```diff
@@ -1,9 +1,20 @@
+/* eslint-disable react/no-unused-state */
 import React from 'react';
+import axios from 'axios';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      currentReviews: '',
```

Comment:
Its not necessary to define currentReview because you'll setState later, but if it helps you keep track, leave it there.